starknet-e2e-test/src/lib.rs

```diff
@@ -1,2 +1 @@
-pub mod ethereum_sandbox;
-pub mod starknet_contract;
+pub mod starknet_sovereign;
```

Comment:
Any reason not to move the whole `starknet_sovereign.rs` content into `lib.rs`?
We do this in order to support `starknet_validity.rs` in the future? (I'm fine with that)